tools/orca/Cargo.toml

```diff
@@ -27,10 +27,14 @@ hashbrown = { workspace = true }
 kanidm_client = { workspace = true }
 kanidm_proto = { workspace = true }
 mathru = { workspace = true }
+ndarray = "0.15.6"
 rand = { workspace = true }
 rand_chacha = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
 serde_json = { workspace = true }
+statrs = "0.16.0"
+strum = "0.26.2"
```

Comment:
What are these being added for?

Reply:
statrs for the Multinomial distribution and strum to count enum variants

Reply:
I think we should remove these for now.